assignments/arrays.js

```diff
@@ -62,35 +62,61 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
+// The dealer needs the information on the last car in their inventory.
+//   What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`car is a *1999* *lincoln* *Town Car*`);
 
 // ==== Challenge 3 ====
-// The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
+// The marketing team wants the car models listed alphabetically on the website. 
+// Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for(let i= 0; i < inventory.length; i++){
+carModels.push(inventory[i].carModel);
+}
+console.log(carModels.sort);
+
+// "300M", "4000CS Quattro", "525", "6Series", "Accord", "Aerio", "Bravada", 
+// "Camry", "Cavalier", "Ciera", "Defender Ice Edition", "E-Class", "Econoline E250",
+//  "Escalade", "Escort", "Esprit", "Evora", "Express 1500", "Familia", "Fortwo", "G35",
+//   "GTO", "Galant", "Intrepid", "Jetta", "LSS", "MR2", "Magnum", "Miata MX-5", "Montero Sport", "Mustang",
+//    "Navigator", "Prizm", "Q", "Q7", "R-Class", "Ram Van  3500", "Ram Van 1500", "Sebring", "Skylark", "TT", "Talon",
+//     "Topaz", "Town car", "Windstar", "Wrangler", "Wrangler", "XC70", "Yukon", "riolet"
 
 // ==== Challenge 4 ====
-// The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
+// The accounting team needs all the years from every car on the lot. 
+// Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for(let i = 0; i < inventory.length; i++)
+carYears.push(inventory[i].car_year)
+console.log(carYears);
+
+// [2009, 2001, 2010, 1983, 1990, 1995, 2009, 1987, 1996, 2000, 2004, 2004, 1997, 1999, 2000, 2001, 1987, 1995, 1994, 1985, 2003, 1997, 1992, 2003, 2005, 2005, 2000, 2005,
+//  1993, 2010, 1964, 1999, 2011, 1991, 2000, 2003, 1997, 1992, 1998, 2012, 1965, 1996, 2009, 2012, 2008, 1995, 2007, 2008, 1996, 1999]
 
 // ==== Challenge 5 ====
-// The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars =[];
-console.log(); 
+// The car lot manager needs to find out how many cars are older than the year 2000. 
+// Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array
+//  oldCars and logging it's length.
+let oldCars =['1983', '1990', '1995', '1987', '1996', '1997', '1999', '1987', '1995', '1994', '1985', '1997', '1992', '1993', '1964', '1999',
```

Comment:
you are supposed to iterate over the inventory to extract the year not hardcode them.